src/main/java/com/pinterest/secor/transformer/DefaultMessageTransformers.java

```diff
@@ -0,0 +1,29 @@
+package com.pinterest.secor.transformer;
+
+import com.pinterest.secor.common.SecorConfig;
+
+/**
+ * Default message transformer class which does no transformation
+ * 
+ * @author Ashish (ashu.impetus@gmail.com)
+ *
+ */
+public class DefaultMessageTransformers implements MessageTransformers {
```

Comment:
This class should have a more descriptive name, e.g., IdentityMessageTransformer.